augur/tasks/github/pull_requests/tasks.py

```diff
@@ -338,7 +389,6 @@ def collect_pull_request_reviews(repo_git: str, full_collection: bool) -> None:
     tool_source = "pull_request_reviews"
     data_source = "Github API"
 
-    repo_id = get_repo_by_repo_git(repo_git).repo_id
```

Comment:
why was this removed?

Reply:
This was a duplicate call. repo_id is being reset on line 397 later.